deployment/terraform/iam.tf

```
@@ -68,6 +68,31 @@ resource "aws_iam_role_policy" "batch_describe_and_submit" {
   policy = data.aws_iam_policy_document.batch_describe_and_submit.json
 }
 
+
+data "aws_iam_policy_document" "s3_read_write_files_bucket" {
+  statement {
+    effect = "Allow"
+
+    resources = [
+      aws_s3_bucket.files.arn,
+      "${aws_s3_bucket.files.arn}/*",
+    ]
+
+    actions = [
+      "s3:DeleteObject",
+      "s3:GetObject",
+      "s3:ListBucket",
+      "s3:PutObject"
+    ]
+  }
+}
+
+resource "aws_iam_role_policy" "s3_read_write_files_bucket" {
+  name   = "S3ReadWriteFiles"
+  role   = aws_iam_role.ecs_task_role.name
```

When I ran `scripts/infra plan` in a terraform container I got an error on this line. Looking at the other policies in the file, I think we need to use `app_task_role`

```suggestion
  role   = aws_iam_role.app_task_role.name
```